packages/workspace/src/core/project-graph/operators.spec.ts

```diff
@@ -171,3 +171,24 @@ describe('withDeps', () => {
     });
   });
 });
+
+describe('filterType', () => {
```

Comment:
This isn't an actual function... can we make it a function?

Reply:
Renamed to `filterNodes`, and also added `onlyWorkspaceProjects` as a convenience function.